src/scenechange/mod.rs

```diff
@@ -200,7 +194,7 @@ impl<T: Pixel> SceneChangeDetector<T> {
 
     if scenecut {
       // Clear buffers and deque
-      self.frame_buffer.clear();
+      self.frame_buffer = None;
```

Comment:
Actually, do we need to clear the frame buffer if a scenecut is found? This is something the original version did, but I don't know if it's actually necessary

Reply:
I think we do (and should) because after a scenecut is found, we're not guaranteed to look at the next frame (because of min scenecut distance). So we need to clear it to make sure the frames in the buffer remain contiguous.